apps/src/lib/cli.rs

```diff
@@ -2035,18 +2041,23 @@ pub mod args {
             app.arg(
                 NAMADA_START_TIME
                     .def()
-                    .about("The start time of the ledger."),
+                    .help("The start time of the ledger."),
             )
-            .arg(BLOCK_HEIGHT.def().about("The block height to run until."))
-            .arg(HALT_ACTION.def().about("Halt at the given block height"))
+            .arg(BLOCK_HEIGHT.def().help("The block height to run until."))
+            .arg(HALT_ACTION.def().help("Halt at the given block height"))
             .arg(
                 SUSPEND_ACTION
                     .def()
-                    .about("Suspend consensus at the given block height"),
+                    .help("Suspend consensus at the given block height"),
             )
             .group(
                 ArgGroup::new("find_flags")
-                    .args(&[HALT_ACTION.name, SUSPEND_ACTION.name])
+                    .args(
+                        #[allow(clippy::needless_borrow)]
+                        {
+                            &[HALT_ACTION.name, SUSPEND_ACTION.name]
+                        },
+                    )
```

Comment:
```suggestion
                    .args([HALT_ACTION.name, SUSPEND_ACTION.name])
```

Reply:
On some reason, this makes my rust-analyzer fail.

Reply:
Reverted 883597cee3d41e265634d8b569baaff439a66672